content/_index.md

```diff
@@ -0,0 +1,21 @@
+# Linux as Firmware
+**Tired of reinventing the wheel by implementing drivers for firmware again and again.**</br>
+**Not with LinuxBoot!**
+
+![](images/linuxboot_info.png)
+
+### What?
+LinuxBoot is a firmware for modern servers that replaces specific firmware functionality
+like the UEFI DXE phase with a Linux kernel and runtime.
+
+### Why?
+* Reduces the attack surface by replacing firmware drivers with Linux.
+* Improves boot time by removing unnecessary code.
+* Allows customization of the initrd runtime to support site specific
+needs (both device drivers as well as custom executables).
+* Provides an environment for security research.
```

Comment:
Research is not a good word to use, I've learned. In fact people tend to react negatively: "this is just a science project". They assume that if you mention research then this approach is not ready or usable.

I'd rather see this replaced with
Proven approach for almost 20 years in Military, Consumer Electronics, and Supercomputing systems --whereever reliability and performance are paramount

If you want to mention security I think it should be in the context of providing security for real world uses.